bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -459,23 +463,21 @@ def predict(self):
                     self._candidates[class_name].append(c)
 
 
-class YoloBallDetector(CandidateFinder):
+class YoloDetector(CandidateFinder):
     """
-    A ball detector using the yolo neural network.
+    An abstract object detector using the yolo neural network.
     This layer connects a single YOLO network with multiple candidate finders for the different classes,
-    in this case the ball class.
+    in this case the goalpost class.
     """
     def __init__(self, config, yolo):
         """
-        Constructor for the YoloBallDetector.
+        Constructor for the YoloGoalpostDetector.
```

Comment:
Isn't this the general yolo detector? Thus it shouldn't say goalpost?

Reply:
Fixed it